servers/src/proto/journal/journal.proto

```diff
@@ -17,9 +17,6 @@ message JournalEntry {
     BlockContainerIdGeneratorEntry block_container_id_generator = 3;
     BlockInfoEntry block_info = 4;
     CompleteFileEntry complete_file = 5;
-    CompletePartitionEntry complete_partition = 21;
```

Comment:
@gpang @apc999 Do you think it makes more sense to move the key value entries separate from the file entries?